samples/csharp_dotnetcore/16.proactive-messages/Bots/ProactiveBot.cs

```diff
@@ -10,7 +10,9 @@
 
 namespace Microsoft.BotBuilderSamples
 {
+#pragma warning disable CA1724 // Type names should not match namespaces (by design and we can't change this without breaking binary compat).
```

Comment:
Check these errors. In this case, it's related to the namespace that the `NotifyController` class has. If you fix the namespace in that class to match the rest of the bot's files, the error disappears.

Reply:
Got it. I think we should use something like **Microsoft.BotBuilderSamples.Controllers**, which is what we used in most of the controllers.